src/Symfony/Bundle/FrameworkBundle/Resources/config/error_renderer.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services https://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <defaults public="false" />
+
+        <service id="error_handler.error_renderer" class="Symfony\Component\ErrorHandler\DependencyInjection\ErrorRenderer">
+            <argument /> <!-- error renderer locator -->
+        </service>
+
+        <service id="error_handler.renderer.html" class="Symfony\Component\ErrorHandler\ErrorRenderer\HtmlErrorRenderer">
+            <tag name="error_handler.renderer" />
+            <argument>%kernel.debug%</argument>
+            <argument>%kernel.charset%</argument>
+            <argument>%debug.file_link_format%</argument>
+        </service>
+
+        <service id="error_handler.renderer.json" class="Symfony\Component\ErrorHandler\ErrorRenderer\JsonErrorRenderer">
+            <tag name="error_handler.renderer" />
+            <argument>%kernel.debug%</argument>
+        </service>
+
+        <service id="error_handler.renderer.xml" class="Symfony\Component\ErrorHandler\ErrorRenderer\XmlErrorRenderer">
+            <tag name="error_handler.renderer" format="atom" />
```

Comment:
I don't think this works. The final format the renderer is registered by is using the static method of the class which is xml, see https://github.com/symfony/symfony/blob/b92e4ed9d4e480ecfb5dfc0f1dd588b283c7e0a6/src/Symfony/Component/ErrorCatcher/ErrorRenderer/ErrorRenderer.php#L38
So the "atom" format is not available.

Reply:
See `ErrorCatcherPass` it uses the `Symfony\Component\ErrorCatcher\DependencyInjection\ErrorRenderer` instead.

Reply:
#32232